includes/Core/Admin/Available_Tools.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Admin\Available_Tools
+ *
+ * @package   Google\Site_Kit\Core\Admin
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Admin;
+
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Util\Method_Proxy_Trait;
+use Google\Site_Kit\Core\Util\Reset;
+
+/**
+ * Class for extending available tools for Site Kit.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Available_Tools {
+	use Method_Proxy_Trait;
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+		add_action( 'tool_box', $this->get_method_proxy( 'render_tool_box' ) );
+	}
+
+	/**
+	 * Renders tool box output.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function render_tool_box() {
+		if ( ! current_user_can( Permissions::SETUP ) ) {
+			return;
+		}
+		$reset_url = add_query_arg(
+			array(
+				'action' => Reset::ACTION,
+				'nonce'  => wp_create_nonce( Reset::ACTION ),
+			),
+			admin_url( 'index.php' )
+		);
```

Comment:
Let's probably move it to a new public static method `Reset::get_reset_link`, so we can reuse it to generate reset links.

Reply:
I thought about this too and I agree. I was thinking maybe `Reset_Persistent` shouldn't have this method but we can always overload it there. Otherwise, it's fine if they both point to the same URL for now.